src/ConformalTracking.cc

```diff
@@ -1444,7 +1444,7 @@ void ConformalTracking::extendTracks(std::vector<KDTrack*>& conformalTracks, std
 
 // Extend seed cells
 void ConformalTracking::extendSeedCells(std::vector<Cell*>& cells, KDTree* nearestNeighbours, bool extendingTrack,
-                                        std::vector<KDCluster*> debugHits) {
+                                        const std::vector<KDCluster*>& /*debugHits*/) {
```

Comment:
What does this notation /*debugHits*/ mean?

Reply:
`/*debugHits*/` just comments the parameter name because it is unused